apps/src/applab/applab.js

```diff
@@ -1154,9 +1240,9 @@ studioApp.reset = function(first) {
   divApplab.parentNode.replaceChild(newDivApplab, divApplab);
 
   divApplab = document.getElementById('divApplab');
-  if (Applab.levelHtml) {
-    divApplab.innerHTML = Applab.levelHtml;
-  }
+  var isDesignMode = window.$ && $('#codeModeButton').is(':visible');
+  Applab.parseFromLevelHtml(divApplab, isDesignMode);
+
   divApplab.addEventListener('click', Applab.onDivApplabClick);
```

Comment:
After investigating further: We clone the original element to preserve the width, height and tabindex attributes. We do not put the swap after parseFromLevelHtml because the divApplab must be in the document already in order for parseFromLevelHtml to make its children draggable. 

I've removed step 5 per your suggestion. @islemaster , based on these answers, are there additional changes you would still like to see?